src/test/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator has started work.");
```

Comment:
The message should be more informative and indicate that it is an Excavator that has started working.